docs/faq.rst

```diff
@@ -280,6 +280,40 @@ I'm scraping a XML document and my XPath selector doesn't return any items
 
 You may need to remove namespaces. See :ref:`removing-namespaces`.
 
+SSL Errors on OS X
+------------------
+
+At the time of this writing (April 2016), the version of OpenSSL that
+ships on OS X El Capitan (10.11.4) is quite old. The system OpenSSL is
+version ``0.9.8zg`` whereas the latest version is ``1.0.2g``.
+
+The current version of Scrapy (1.0.5) also has issues that will
+be addressed in version 1.1, so we'll need to install the release
+candidate.
```

Comment:
Well I'd say we're at a bit of a catch-22 then. Since the current version is broken, the next one is not but to publish this guide you need a new version.

Merge it or not or edit it or not.